tests/cases/fourslash/completionListForImportAttributes.ts

```diff
@@ -0,0 +1,20 @@
+///<reference path="fourslash.ts" />
+
+// @module: esnext
+// @target: esnext
+// @filename: ./a.ts
+////export default {};
+
+// @filename: ./b.ts
+////declare global {
+////    interface ImportAttributes {
+////        type: "json",
+////        "resolution-mode": "import"
+////    }
+////}
+////
+////import * as t1 from "./a" with { /*1*/ };
+////import * as t2 from "./a" with { type: "/*2*/" };
+////import * as t3 from "./a" with { type: "json", /*3*/ };
+
+verify.baselineCompletions();

```

Comment:
It might be nice to show that locals/globals are excluded when the contextual type allows any `string`, since values have to be literals, not references:

```ts
declare global {
  interface ImportAttributes {
    type: string
  }
}

const str = "hello";
import * as t4 from "./a" with { type: /* exclude 'str' */ };
```